src/annotator/components/Toolbar.tsx

```diff
@@ -146,8 +146,22 @@ export default function Toolbar({
   return (
     <div
       className={classnames(
-        'absolute left-[-33px] w-[33px] z-2',
+        {
+          // For minimal controls, display the toolbar to the left, fully
+          // outside the sidebar
+          'absolute right-full': useMinimalControls,
+          // When the full toolbar is displayed, we position it relative to its
+          // container, so that it takes vertical space and pushes next elements
+          // down (eg. the buckets list).
+          // We push it away from the right side to compensate for the parent
+          // hardcoded width.
```

Comment:
```suggestion
          // container, so that it takes vertical space and pushes next elements
          // down (eg. the buckets list).
          // The toolbar is wider than its parent, so we need to adjust the right position
          // so that the right edge of the toolbar aligns with the right edge of the parent.
```

I suggested a slightly rewording here to describe the intended effect, which is going to be more durable than how it is implemented.